cluster-scope/overlays/nerc-ocp-prod/kustomization.yaml

```diff
@@ -13,5 +13,6 @@ resources:
 - apiserver/cluster.yaml
 - secretstores
 
-patches:
-- path: ingresscontrollers/default_patch.yaml
```

Comment:
By looking at the documentation, it seems that when using `patches` you need to specify both `path` and `target`, which wasn't the case here. So I revised this to just be a strategic merge.